src/bindings/c/tests/ie_c_api_test.cpp

```diff
@@ -131,7 +135,11 @@ TEST(ie_core_register_plugin, registerPlugin) {
     IE_EXPECT_OK(ie_core_read_network(core, xml, bin, &network));
     EXPECT_NE(nullptr, network);
 
-    const char *plugin_name = "openvino_intel_cpu_plugin";
+    #ifdef __aarch64__
+        const char *plugin_name = "openvino_arm_cpu_plugin";
```

Comment:
I think that plugin name can be a common definition for this file.

Reply:
how about this?
```
#define ARM_CPU_PLUGIN_NAME "openvino_arm_cpu_plugin"
#define INTEL_CPU_PLUGIN_NAME "openvino_intel_cpu_plugin"
```

Reply:
Why not something like:
```suggestion
#ifdef __aarch64__
    #define CPU_PLUGIN_NAME "openvino_arm_cpu_plugin"
#else
    #define CPU_PLUGIN_NAME "openvino_intel_cpu_plugin"
#endif
```

Reply:
Good idea, will do this change. Thanks!